pkg/collector/osmlogs_collector.go

```diff
@@ -67,21 +67,36 @@ func (collector *OsmLogsCollector) Collect() error {
 	if err != nil {
 		return err
 	}
-
+	resources := []string{"configmap", "serviceaccount", "service", "endpoint", "ingress"}
 	for _, meshName := range meshList {
 		namespacesInMesh, err := getResourceList("namespaces", "openservicemesh.io/monitored-by="+meshName, "-o=jsonpath={..name}", " ")
 		if err != nil {
 			return err
 		}
-		collectDataFromNamespaces(collector, namespacesInMesh, rootPath, meshName)
+
 		collectControllerLogs(collector, rootPath, meshName)
+		if err != nil {
+			return err
+		}
```

Comment:
I'd remove these lines - we have no new `err` coming from `collectControllerLogs()` so this would be the exact same as at https://github.com/SanyaKochhar/aks-periscope/pull/8/files#diff-67f7de8f224f5909c6a17771ac59fb3982c68e0d3e8a16c42017c83138482428R73

Reply:
@draychev Oops, I meant to propagate the error for collectControllerLogs so it should be
```
err = collectControllerLogs(collector, rootPath, meshName)
if err != nil {
	return err
}
```